site/client/css/components/sparkbars.scss

```diff
@@ -0,0 +1,125 @@
+$table-bg: $body-bg;
+.map-tooltip {
+    .time-series-value {
+        line-height: 1.3;
+        white-space: nowrap;
+
+        .count {
+            font-size: 16px;
+            font-weight: 700;
+            display: block;
+            margin-bottom: 1px;
+        }
+
+        .date {
+            font-size: 13px;
+            font-weight: 400;
+            display: block;
+            opacity: 1;
+        }
+
+        .date.latest {
+            font-size: 12px;
+            font-weight: 400;
+            opacity: 0.65;
+        }
+
+        .help-icon {
+            font-size: 15px;
+            color: rgba($primary-color, 0.3);
+            transition: color 200ms ease;
+
+            &:hover {
+                color: rgba($primary-color, 0.65);
+            }
+        }
+
+        $bar-color: #ab0000;
+        $highlight-color: $bar-color;
+
+        .time-series-value.current .count {
+            color: $bar-color;
+        }
+
+        .time-series-value.highlighted {
+            color: $highlight-color;
+        }
+
+        .bar {
+            background-color: rgba(desaturate($bar-color, 40%), 0.25);
+            &.current {
+                background-color: $bar-color;
+            }
+            &.highlighted {
+                background-color: $highlight-color;
+            }
+        }
+    }
+
+    .trend {
+        display: flex;
+        height: 100%;
+
+        .plot {
+            flex: 0;
+            margin-bottom: -1px;
+            align-self: flex-end;
+        }
+
+        .value {
+            flex: 1;
+            align-self: center;
+            padding-top: 10px;
+            padding-bottom: 10px;
+            padding-left: 6px;
+        }
+    }
+}
+
+.covid-table-mobile {
```

Comment:
More general comment: I don't know where those styles were located before, but it's probably good to remove the old ones.